cpu/stm32/periph/spi.c

```diff
@@ -162,7 +206,17 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
     /* enable SPI device clock */
     periph_clk_en(spi_config[bus].apbbus, spi_config[bus].rccmask);
     /* enable device */
-    uint8_t br = spi_divtable[spi_config[bus].apbbus][clk];
+    //if (clk != clocks[bus]) {
```

Comment:
Why remove the caching?

Reply:
Because it is getting late and I wanted to measure the actual impact of the code. :sleeping:

Reply:
Restored